src/utils.rs

```diff
@@ -0,0 +1,8 @@
+pub(crate) fn platform() -> crate::Result<String> {
+    let output = std::process::Command::new("uname").arg("-a").output()?;
```

Comment:
Make this return some fixed string in test mode. Same as client_version. Else we will have to keep updating files in tests folder.